src/com/goide/runconfig/GoRunConfigurationWithMain.java

```diff
@@ -38,11 +38,9 @@
 public abstract class GoRunConfigurationWithMain<T extends GoRunningState> extends GoRunConfigurationBase<T> {
   private static final String FILE_PATH_NAME = "file_path";
   
-  @NotNull private String myFilePath = "";
-
   public GoRunConfigurationWithMain(String name, GoModuleBasedConfiguration configurationModule, ConfigurationFactory factory) {
     super(name, configurationModule, factory);
-    myFilePath = getWorkingDirectory();
+    setFilePath(getWorkingDirectory());
```

Comment:
You have defined the field in base class, so store the field there and remove storing duplicates in concrete run configurations